### PR TITLE
update docker documentation

### PR DESCRIPTION
**Pull request by VictorBarbosaMartins**
Quick update to our docker documentation such that we (or I) don't forget anymore to update the container image.

Added this only to the developer container but useful for both.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -86,6 +86,14 @@ docker run --rm -it -v "$(pwd)/:/workdir/external" ghcr.io/gammasim/simtools-dev
 
 Remember you need to `docker login` to the GitHub package repository with a personal token in order to download an image (follow [these instructions](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)).
 
+Note: if the docker image already exists in your system, this same image will be used to run the container. Please make sure to update the image regularly by, e.g., removing it **before** running the command above. This can be done by passing the docker id:
+
+```bash
+docker rmi <Your docker ID>
```
Comment (GernotMaier) [inline docker/README.md:92]: "your docker ID" is maybe not entirely clear, as it sounds like a personal ID.  Please call id 'image ID".

**Review by GernotMaier — APPROVED** [review]
One small suggestion to improve. No need to revisit, so I approve already.

File: docker/README.md (hunk 1 above)
Comment (VictorBarbosaMartins) [reply docker/README.md]: agreed, done.

**Comment by VictorBarbosaMartins** [comment]
Thanks for checking and approving the PR

**Pull request merged**